client/my-sites/ads/form-earnings.jsx

```diff
@@ -289,7 +289,7 @@ class AdsFormEarnings extends Component {
 							<tr>
 								<th className="ads__earnings-history-header">{ translate( 'Period' ) }</th>
 								<th className="ads__earnings-history-header">{ translate( 'Earnings' ) }</th>
-								<th className="ads__earnings-history-header">{ translate( 'Ad Impressions' ) }</th>
+								<th className="ads__earnings-history-header">{ translate( 'Attempted Impressions' ) }</th>
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'Impressions: ' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/ad-earnings-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).